astropy/coordinates/funcs.py

```diff
@@ -159,9 +159,9 @@ def get_sun(time):
     # convert barycentric velocity to units of c, but keep as array for passing in to erfa
     earth_v /= c.to(u.au/u.d).value
 
-    dsun = np.sqrt(np.sum(earth_p**2, axis=-1, keepdims=True))
+    dsun = np.sqrt(np.sum(earth_p**2, axis=-1))
     invlorentz = (1-np.sum(earth_v**2, axis=-1))**0.5
```

Comment:
@eteq - Your change was indeed exactly what I tried, and I did run the test by hand, but in doing that I must somehow have not actually tested that version (I have noticed that sometimes when I install something in a virtual environment, I still get the system astropy version; I'll have to look into that...).

Anyway, sorry for the distraction...